indicator/src/main/java/com/fuzz/indicator/LayeredImageViewHolder.java

```diff
@@ -31,6 +32,6 @@ public LayeredImageViewHolder(ImageView itemView) {
 
     @Override
     public void offsetImageBy(int orientation, float percentage) {
-        OffSetters.offsetImageBy(itemView, orientation, percentage);
+        OffSetters.offsetImageBy(itemView, orientation, percentage, new Matrix());
```

Comment:
I think for now, the Matrix should be the concern of the LayeredImageViewHolder. If we ever need to change how it's stored, that's what should get first say and that's what should need to be modified - not OffSetters.